### ISSUE TITLE
filter_var with FILTER_VALIDATE_IP is not working

### ISSUE THREAD

**Issue by calvinbaart** [peachpiecompiler/peachpie#481]
Sample:
```php
$tests = [
    ["127.0.0.1", FILTER_VALIDATE_IP],
    ["invalid-ip", FILTER_VALIDATE_IP],
    ["127.0.0.1", FILTER_VALIDATE_IP, FILTER_FLAG_IPV4],
    ["::1", FILTER_VALIDATE_IP, FILTER_FLAG_IPV4],
    ["invalid-ip", FILTER_VALIDATE_IP, FILTER_FLAG_IPV4],
    ["127.0.0.1", FILTER_VALIDATE_IP, FILTER_FLAG_IPV6],
    ["::1", FILTER_VALIDATE_IP, FILTER_FLAG_IPV6],
    ["invalid-ip", FILTER_VALIDATE_IP, FILTER_FLAG_IPV6],
];

foreach ($tests as $index => $test) {
    $result = filter_var($test[0], $test[1], $test[2] ?? null);

    if ($result) {
        echo "{$index} => {$result}\r\n";
    } else {
        echo "{$index} => false\r\n";
    }
}
```

PeachPie output:
```
0 => false
1 => false
2 => false
3 => false
4 => false
5 => false
6 => false
7 => false
```

PHP output:
```
0 => 127.0.0.1
1 => false
2 => 127.0.0.1
3 => false
4 => false
5 => false
6 => ::1
7 => false
```

**Comment by jakubmisek** [comment]
IP filter was not implemented, we should report the PHP warning about invalid argument more obviously :)
